dev-infrastructure/configurations/mvp-mgmt-cluster.bicepparam

```diff
@@ -6,6 +6,7 @@ param vnetAddressPrefix = '10.132.0.0/14'
 param subnetPrefix = '10.132.8.0/21'
 param podSubnetPrefix = '10.132.64.0/18'
 param enablePrivateCluster = false
+param aksClusterName = take('aro-hcp-mgmt-cluster-${uniqueString('mgmt-cluster')}', 63)
```

Comment:
i would like to keep the names backwards compatible for the MVP SC and MC for now. we can beautify the name next time we may need to reinstall them.